packages/wdio-spec-reporter/src/index.js

```diff
@@ -44,11 +45,15 @@ class SpecReporter extends WDIOReporter {
     }
 
     onTestPass () {
+        this.stateCounts.prevPassed = this.stateCounts.passed
         this.stateCounts.passed++
     }
 
     onTestFail () {
         this.stateCounts.failed++
+        if (this.stateCounts.prevPassed == this.stateCounts.passed - 1) {
```

Comment:
always `===` in JS
```suggestion
        if (this.stateCounts.prevPassed === this.stateCounts.passed - 1) {
```

Reply:
Sorry didn't realise I could just commit suggestion - I've pushed a commit implementing it!